lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py

```diff
@@ -1,12 +1,9 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-# Copyright: (c) 2019, Ansible Project
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
-
-from __future__ import absolute_import, division, print_function
-
-__metaclass__ = type
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
```

Comment:
Not sure if we can license module code as Apache License.

Reply:
cc @abadger @thaumos

Reply:
Ah, will change back. GPL